troposphere/views/guacamole.py

```diff
@@ -0,0 +1,87 @@
+import json
```

Comment:
Hrm, I wasn't aware that we had the other views for web_desktop/vnc in troposphere. Wish that stuff had been done in atmosphere. Is there a good reason to implement these views in troposphere, that I'm not aware of?